src/miv_simulator/simulator/_generate_input_features.py

```diff
@@ -40,7 +40,7 @@ def mpi_excepthook(type, value, traceback):
 
```

Comment:
I see the import problem here, but instead of making the modules private, I think it might be better to rename them to something that does not clash, like `distribute_synapse_locations` -> `synapse_locations` etc.

Reply:
Yeah. We can probably refactor all of these when we determine what to do with the executable codes in `scripts`.

Reply:
Sounds good. I should have done that in #10 🙈

Reply:
Its all good. I don't think we are fully decided yet.